content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -36,9 +36,14 @@ that can be attached to a Node:
 
 ## Custom limits
 
-You can change these limits by setting the value of the
+You can change the limits for in-tree volumes by setting the value of the
 `KUBE_MAX_PD_VOLS` environment variable, and then starting the scheduler.
 
+It is recommended that CSI drivers allow for customization of volume limits
+as well. That way cluster administrators can distribute the limits of the
+same storage backends (e.g. iSCSI) accross different drivers, according to
+their individual needs.
```

Comment:
@gnufied can you take a look at this?

Reply:
I think this should be documented in - https://kubernetes-csi.github.io/docs/csi-node-object.html and not here because these are mostly user facing docs.

Reply:
Makes sense. Removed